### ISSUE TITLE
regression with #820

### ISSUE THREAD

**Issue by krimdomu** [RexOps/Rex#887]
quote ferki: (see #820)

Ha! I found a regression caused by this commit: for some reason it breaks the Rex::Test features, more specifically rex can't create test VMs anymore with Entity: line 2: parser error : Premature end of data in tag domain line 1 or similar errors.

**Comment by krimdomu** [comment]
@ferki 
this works for me (tested on windows locally with virtual box, on windows remote on a linux system with kvm and on a linux system remote on a linux system with kvm)

could you please verify this?

**Comment by ferki** [comment]
Sure, I'll test it again.

**Comment by ferki** [comment]
Unfortunately I still get these errors:
```
$ rex -qw Test:run --test=t/ssh.t
[2016-01-21 14:15:31] <local> ERROR - Error running t/ssh.t: Entity: line 1: parser error : Premature end of data in tag host line 1
7:+77</baselabel>      <baselabel type='qemu'>+77:+77</baselabel>    </secmodel>
                                                                                ^
Entity: line 1: parser error : Premature end of data in tag capabilities line 1
7:+77</baselabel>      <baselabel type='qemu'>+77:+77</baselabel>    </secmodel>
                                                                                ^
XML::Simple called at /home/ferki/perl5/perlbrew/perls/perl-5.22.0/lib/site_perl/5.22.0/Rex/Virtualization/LibVirt/hypervisor.pm line 39.
```

**Comment by ferki** [comment]
I think that's because `Rex::Interface::Exec::Local` was left unmodified (i.e. not using `sysread`).

**Comment by ferki** [comment]
Fixed on #908.
